lib/di.dart

```diff
@@ -438,6 +440,19 @@ Future<void> setup({
     }, closable: false);
   }, instanceName: 'login');
 
+  getIt.registerSingleton<BottomSheetService>(BottomSheetServiceImpl());
+
+  final appStore = getIt.get<AppStore>();
+  getIt.registerLazySingleton<WalletConnectService>(() {
```

Comment:
this would only be initialized if the user opened the connection and sync page.
what if I already connected to a dApp and closed Cake and then re-opened it and then triggered a transaction from the dApp, I believe in this scenario the app won't show any proposals, since the wallet connect service is not yet initialized and the listeners are not configured, is that correct?

Reply:
I'm thinking we can move the `walletConnectService.setUpWalletConnect()` call to be called around the DashboardPage, that way when a wallet is loaded, it makes the call and initialises the service

What do you think?